demos/demo-react-ts/src/components/App.tsx

```diff
@@ -51,12 +69,16 @@ function App() {
   const [showAlert, setShowAlert] = useState(true);
   const [fromNumber, setFromNumber] = useState("+1 617-948-3986");
 
+  const [availability, setAvailability] = useState<Availability>("UNAVAILABLE");
```

Comment:
Since availability is a boolean, we could simply use a boolean to simplify the state
```
const [userAvailable, setUserAvailable] = useState(false)
```

Reply:
I agree with your point. I, think, if we have string literals, its easy to read and extend should we need.